lib/practice_exercises.rb

```diff
@@ -1,13 +1,68 @@
-
-# Time Complexity: ?
-# Space Complexity: ?
+require 'pry'
+# Time Complexity: O(n) - as array size grows, the evaluation grows at the same rate 
```

Comment:
Since `delete_at` shifts all subsequent letters over one value to the left, and has a time complexity of O(n), your algorithm is O(n)